Max.java

```diff
@@ -5,9 +5,19 @@ static int max(int a, int b){
 		int max=a>b?(a):(b);
 		return max;
 	}
+// 	static int max(int a, int b){

```

Comment:
This could be another way of finding maximum using if else but ternary operators are short and easy to understand